### PR TITLE
fix(scrollbar): Make sidebar scrollbar nicer

### PR DESCRIPTION
**Pull request by vgrozdanic**
Scrollbar on sidebar looks quite ugly compared to the rest of the UI. This PR is a fix to make the scrollbar looks nicer on Chrome and Firefox browsers. [Safari browser doesn't support used CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color)

Current look
image

New look after changes
image

### CODE REVIEW

**Review by ddevsr — APPROVED** [review]

**Review by natemoo-re — APPROVED** [review]
LGTM!

File: static/app/utils/theme.tsx [hunk 1]
```diff
@@ -971,6 +971,9 @@ export const lightTheme = {
   sidebar: {
     ...commonTheme.sidebar,
     background: sidebarBackground.light,
+    scrollbarWidth: 'thin',
+    scrollbarThumbColor: '#A0A0A0 ',
```
Comment (ArthurKnaus) [inline static/app/utils/theme.tsx:975]:
```suggestion
    scrollbarThumbColor: '#A0A0A0',
```

File: static/app/utils/theme.tsx [hunk 2]
```diff
@@ -999,6 +1002,9 @@ export const darkTheme: Theme = {
   sidebar: {
     ...commonTheme.sidebar,
     background: sidebarBackground.dark,
+    scrollbarWidth: 'thin',
+    scrollbarThumbColor: '#808080 ',
```
Comment (ArthurKnaus) [inline static/app/utils/theme.tsx:1006]:
```suggestion
    scrollbarThumbColor: '#808080',
```

**Review by ArthurKnaus — APPROVED** [review]
🙌

File: static/app/utils/theme.tsx (hunk 2 above)
Comment (vgrozdanic) [reply static/app/utils/theme.tsx:1006]: Good eye, thanks for catching this 😓

**Pull request merged**